daemon/authorization/spotify.py

```diff
@@ -0,0 +1,57 @@
+from io import BytesIO
+from urllib.parse import urlencode
+import pycurl
+import base64
+import json
+
+
+class Spotify:
+
+    """
+        To make the API fast for everybody, rate limits apply.
+        Unauthenticated requests are processed at the lowest rate limit.
+        Authenticated requests with a valid access token benefit from higher rate limits
+    """
+
+    # Spotify Authorize API
+    spotify_authorize_url = "https://accounts.spotify.com/api/token"
+
+    def __init__(self, client_id, client_secret, grant_type="client_credentials"):
+        """
+            Constructor
+            Parameters:
+                client_id - provider by Spotify when register app. (Required)
+                cleint_secret - provider by Spotify when register app. (Required)
+                grant_type - grant type, set it to “client_credentials”. (Required)
+        """
```

Comment:
Since you give the `grant_type` `client_credentials` as the default value, this is not the `required` anymore. It becomes to `option`. Anyway if you want to write down the comment, it should be right. ;)
Once @cclien0725 defines the coding convention, we should follow his convention to write down the comments.